enterprise/reporting/error_extractor.go

```diff
@@ -109,48 +109,77 @@ func (ext *ExtractorHandle) getSimpleMessage(jsonStr string) string {
 	}
 
 	for key, erRes := range jsonMap {
-		erResStr, isString := erRes.(string)
+		if result := ext.handleKey(key, erRes); result != nil {
+			return *result
+		}
+	}
+	return ""
+}
+
+func (ext *ExtractorHandle) handleKey(key string, value interface{}) *string {
+	valueStr, isString := value.(string)
+	switch key {
+	case "reason", "Error", responseKey, errorKey:
```

Comment:
We can move `value.(string)` within the case block. 
```suggestion
func (ext *ExtractorHandle) handleKey(key string, value interface{}) *string {
	switch key {
	case "reason", "Error", responseKey, errorKey:
		valueStr, isString := value.(string)
```